### PR TITLE
Alerts component

### PR DESCRIPTION
**Pull request by prath-dev**
Added Alerts Component

Live Preview: https://pdui.netlify.app/#alerts

### CODE REVIEW

File: components/Alerts/alerts.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.alert-msg {
+    display: flex;
+    justify-content: left;
+    height: 57px;
+    width: 522px;
```
Comment (Jayesh3636) [inline components/Alerts/alerts.css:5]: suggestion: try to give use REM to make it resp rather ussing px

Comment (prath-dev) [reply components/Alerts/alerts.css:5]: okk thanks for your suggestion.

**Comment by Jayesh3636** [comment]
If you can, try to make the UI with the proper theme!

File: components/Alerts/alerts.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <title>replit</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link href="alerts.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+    <div class="Alerts-container">
+        <h1>Alerts</h1>
+        <h3>Alerts are used to attract user's attention for important information without interrupting the user's flow.
+        </h3>
```
Comment (anurag-kkumarr) [inline components/Alerts/alerts.html:16]: use semantics html like section, nav etc

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
use section tag below instead of div

File: components/Alerts/alerts.html (hunk 2 above)
Comment (prath-dev) [reply components/Alerts/alerts.html:16]: okkkk i thanks for suggestion i will try to include semantics in projects

**Comment by prath-dev** [comment]
@Jayesh3636  good suggestion i will implement in second version of ecommerce ..

**Pull request merged**